.travis.yml

```diff
@@ -27,26 +27,6 @@ before_install:
 
 jobs:
   include:
-    - env:
```

Comment:
can you rebase on the latest master, there were some changes here and potentially those need to be cleaned up as well

Reply:
ok

Reply:
Now
```
 script:
   - ../.travis/run_tests.sh ${TEST_NAME}
```
at the very bottom can be removed (each test now has its own script in this Travis configuration).
But make sure that CircleCI runs https://github.com/pytorch/glow/commit/5213ee61da6dd81f45aaa6e7621043282c2becc4#diff-d0d6e2bbac2adf318f56a0041f9dc844R17